{{ cookiecutter.project_slug }}/CHANGELOG.rst

```diff
@@ -0,0 +1,24 @@
+Change Log
+----------
+
+..
+   All enhancements and patches to {{ cookiecutter.package_name }} will be documented
+   in this file.  It adheres to the structure of http://keepachangelog.com/ ,
+   but in reStructuredText instead of Markdown (for ease of incorporation into
+   Sphinx documentation and the PyPI description).
+   
+   This project adheres to Semantic Versioning (http://semver.org/).
+.. There should always be an "Unreleased" section for changes pending release.
+Unreleased
+~~~~~~~~~~
+
+*
+
+[{{ cookiecutter.version }}] - {% now 'local' %}
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Added
+_____
+
+* First release on PyPI.
```

Comment:
I do not know if we need this block of code, I think it should be a decision of the plugin developer to add this block of code and also version 0.0.0 in theory does not contain any functionality, so... @morenol

Reply:
Yes, I agree with you, though I think that this is useful in order to keep the same format in all the projects, What you think that we can do in order to achieve that? @Squirrel18

Reply:
We can leave it as is, but removing: "* First release on PyPI." since it does not apply to all projects. @morenol

Reply:
I think we can keep this format: [{{ cookiecutter.version }}] - {% now 'local' %}. @morenol